.github/workflows/docker_management.test-PR.yml

```diff
@@ -0,0 +1,82 @@
+name: Build and test image when PR is raised
+
+# This workflow is triggered when Dockerfile related or github action itself changes are made in a PR
+# The workflow is quite simple - builds and test the image. Release of newer version is done only when PR is merged.
+# Design doc: https://github.com/ARMmbed/mbed-os/blob/master/docs/design-documents/docker_management
+
+on:
+  pull_request:
+    branches: [ master ]
+    paths:
+      - docker_images/mbed-os-env/**
+      - .github/workflows/docker_management.*
+      - requirements.txt
+
+jobs:
+
+  build-container:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        platform: [linux/amd64, linux/arm64]
+
+    steps:
+
+      - 
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - 
+        name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
+      # for PR tests, development branch of blinky is used
+      - 
+        name: Checkout
+        uses: actions/checkout@v2
+        with:
+          repository: ARMmbed/mbed-os-example-blinky
```

Comment:
is it sufficient to test with blinky? should we use more advanced example, cloud example for instance

Reply:
environment for official examples should be same and blinky is our go to application for system tests.